applepay/utils.py

```diff
@@ -54,6 +70,24 @@ def signing_time_is_valid(signature, current_time, threshold):
     unexpected format, inconsistent with the CMS 'ContentInfo' object.
     """
     signing_time = retrieve_signature_signing_time(signature)
+    return valid_signing_time(signing_time, current_time, threshold)
+
+
+def valid_signing_time(signing_time, current_time, threshold):
+    """ Validate that the signing time occured within the current time minus
+    the threshold and the current time.
+
+    Args:
+        signing_time (offset-aware datetime): Offset-aware signing time to
+            validate.
+        current_time (offset-aware datetime): Offset-aware current system time
```

Comment:
I actually took these definitions from the previously existing `signing_time_is_valid` docstring.  Do you have a preference of how this is phrased?